ext/node/resolution.rs

```diff
@@ -922,7 +922,7 @@ impl NodeResolver {
             // emitTrailingSlashPatternDeprecation();
           }
           let pattern_trailer = &key[pattern_index + 1..];
-          if package_subpath.len() > key.len()
+          if package_subpath.len() >= key.len()
```

Comment:
This is the fix - for files with single letter names before the extension this conditional was falsy.